assets/component-slider.css

```diff
@@ -359,6 +363,43 @@ slider-component.slider-component-full-width {
   justify-content: center;
 }
 
+.announcement-bar .slider-button--next {
+  margin-right: -1.6rem;
+  height: 100%;
+}
+
+.announcement-bar .slider-button--prev {
+  margin-left: -1.6rem;
+  height: 100%;
```

Comment:
I had to proceed this way because Yoann wanted me to keep the default height of the announcement bar the same as it used to be. If we set a fix height for the buttons 44px then the minimal height of the announcement bar will be bigger.

Reply:
@YoannJailin after discussing it with Ludo, I have to admit that his argument is quite valuable and important. I remember that we discussed it with you and you wanted to keep the default height of the announcement bar the same as it used to be. But because we added arrow buttons and there is a requirement for accessibility we should set those buttons at least 44px height and the announcement bar is going to have a minimum height of 44px as well.